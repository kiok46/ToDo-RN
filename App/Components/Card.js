import React, { Component } from 'react';
import { Text, View, StyleSheet } from 'react-native';


class Card extends Component {

	render (){
		return (
			<View style={[styles.containerStyle ]}>
				{this.props.children}
			</View>
		);
	}
};

const styles = StyleSheet.create({
	containerStyle: {
		borderWidth: 1,
		borderRadius: 2,
		borderColor: '#ddd',
		borderBottomWidth: 0,
		shadowColor: '#000',
		shadowOffset: { width: 0, height:2 },
		shadowOpacity: 0.1,
		shadowRadius: 3,
		elevation: 1,
		marginTop: -1,
		marginBottom: 10,
		margin: 3
	},
	textStyle: {
		fontSize: 20
	}
});

export default Card;