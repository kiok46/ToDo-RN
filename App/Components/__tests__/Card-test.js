import React from 'react';
import Card from '../Card';
import { shallow } from 'enzyme';


describe('testing Card rendering', () => {
    it('renders as expected', () => {
        const wrapper = shallow(
            <Card />
        )
        expect(wrapper).toMatchSnapshot();
    })
})
