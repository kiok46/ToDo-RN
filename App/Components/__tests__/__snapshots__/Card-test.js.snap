// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Card rendering renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": undefined,
      "style": Array [
        Object {
          "borderBottomWidth": 0,
          "borderColor": "#ddd",
          "borderRadius": 2,
          "borderWidth": 1,
          "elevation": 1,
          "margin": 3,
          "marginBottom": 10,
          "marginTop": -1,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.1,
          "shadowRadius": 3,
        },
      ],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": undefined,
        "style": Array [
          Object {
            "borderBottomWidth": 0,
            "borderColor": "#ddd",
            "borderRadius": 2,
            "borderWidth": 1,
            "elevation": 1,
            "margin": 3,
            "marginBottom": 10,
            "marginTop": -1,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 0,
            },
            "shadowOpacity": 0.1,
            "shadowRadius": 3,
          },
        ],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
